packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.js

```diff
@@ -31,6 +31,7 @@ class MoreInfoBox extends React.Component {
 		}
```

Comment:
General comment: Clicking on two (...) menus in a row results in the second focus not working